src/main/java/apaintus/services/draw/textbox/TextBoxDrawService.java

```diff
@@ -19,10 +19,7 @@ public void draw(GraphicsContext context) {
         
         double fontSize = textbox.getStrokeSize();
 
-        if (textbox.getFillColor().compareTo("0x00000000") == 0)
-            textbox.setFillColor("0x000000ff");
-        
-        context.setFill(Paint.valueOf(textbox.getFillColor()));
+        context.setFill(Paint.valueOf(textbox.getStrokeColor()));
```

Comment:
what happen if textbox is null ?

Reply:
Tu ne serais pas dans le DrawService si la shape est null, non?

Reply:
donc si je comprend bien, le use case c'est que la fonction recoit "this" quasiment tout le temp ?

Reply:
pas quasiment. Tout le temps

Reply:
en theroy alors, tu devrais ne pas etre obliger de passer this, ou bien quelquechose m'echape ?

Reply:
@pspino check TextBox class dans la methode getDrawService()

Reply:
nvm all this then. j'ai mal lu